app/src/pages/review/[resourceType]/[id].tsx

```diff
@@ -34,16 +137,151 @@ export default function CommentPage() {
+                                <Anchor
+                                  target="_blank"
+                                  href="https://build.fhir.org/ig/HL7/cqf-measures/ValueSet-artifact-comment-type.html"
+                                  color="dimmed"
+                                  size="xs"
+                                  sx={{ lineHeight: 1 }}
+                                >
+                                  Artifact Comment Types
+                                </Anchor>
+                              </Stack>
+                            </Group>
+                            <Space h="lg" />
+                            <List spacing="sm" size="sm" center>
+                              <List.Item>
+                                <i>documentation:</i> The comment is providing additional documentation from an
+                                authoring perspective
+                              </List.Item>
+                              <List.Item>
+                                <i>review:</i> The comment is providing feedback from a reviewer and requires resolution
+                              </List.Item>
+                              <List.Item>
+                                <i>guidance:</i> The comment is providing usage guidance to an artifact consumer
+                              </List.Item>
+                            </List>
+                          </HoverCard.Dropdown>
+                        </HoverCard>
+                      </Group>
+                    }
+                    icon={<Star opacity={0.5} />}
+                    placeholder="Type"
+                    data={[
+                      { value: 'documentation', label: 'documentation' },
+                      { value: 'guidance', label: 'guidance' },
+                      { value: 'review', label: 'review' }
+                    ]}
+                    {...form.getInputProps('type')}
+                  />
+                </Group>
+                <Textarea
+                  radius="lg"
+                  mt="md"
+                  minRows={15}
+                  maxRows={15}
+                  placeholder="Your Artifact comment"
+                  label="Artifact Comment"
+                  description="Add a comment to the artifact"
+                  withAsterisk
+                  {...form.getInputProps('comment')}
+                />
+                <Space h="md" />
+                <Group grow>
+                  <TextInput radius="lg" label="Endorser Name" placeholder="Name" {...form.getInputProps('name')} />
+                  <Checkbox
+                    ref={ref}
+                    id="checkbox"
+                    color="white"
+                    mt="md"
+                    label="Include Date in Comment"
+                    {...form.getInputProps('date')}
+                    onChange={() => {
+                      setDateSelected(!dateSelected);
+                    }}
+                  />
```

Comment:
Not something that necessarily needs to be changed right now, but I'm having a hard time thinking of a use case where the reviewer would choose not to have the date recorded. Maybe we should bring this up in slack to get some input from Hoss/Paul/others who are more familiar with the measure development process

Reply:
Great point! Just made a thread in slack regarding this!